float64[12] position_x
float64[12] position_y
float64[12] position_z
float64[12] orientation_x
float64[12] orientation_y
float64[12] orientation_z
float64[12] orientation_w
float64 action_number
int32 target_point_number
